compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3850,7 +3850,56 @@ TR::Register *OMR::Power::TreeEvaluator::vdsqrtEvaluator(TR::Node *node, TR::Cod
 
 TR::Register* OMR::Power::TreeEvaluator::vfmaEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   TR::DataType type = node->getDataType().getVectorElementType();
+   if (type != TR::Float && type != TR::Double)
+   {
+      TR_ASSERT(false, "unsupported vector type %s\n", type.toString());
+      return NULL;
+   }
+
+   TR::InstOpCode::Mnemonic op;
+
+   TR::Node *firstChild  = node->getFirstChild();
+   TR::Node *secondChild = node->getSecondChild();
+   TR::Node *thirdChild  = node->getThirdChild();
```

Comment:
I think a comment here indicating which child refers to which part of the fma would be helpful here.